tests/avifyuv.c

```diff
@@ -313,7 +319,11 @@ int main(int argc, char * argv[])
             avifRGBImage srcRGB;
             avifRGBImageSetDefaults(&srcRGB, image);
             srcRGB.depth = yuvDepth;
-            avifRGBImageAllocatePixels(&srcRGB);
+            if (avifRGBImageAllocatePixels(&srcRGB) != AVIF_RESULT_OK) {
+                avifImageDestroy(image);
+                printf("ERROR: Out of memory\n");
```

Comment:
cases like this already exist, so it's fine for consistency, but I think the tests should output all errors to stderr.